PSKoans/Koans/Constructs and Patterns/AboutBitwiseOperations.Koans.ps1

```diff
@@ -0,0 +1,546 @@
+            <#
+                Bits will fall out of scope when shifting to the right in a similar manner as shifting left.
+
+                When a Byte has the value 1, and the value is shifted one bit to the right the result is 0.
+
+                Shifting the lowest order bit of a numeric value to the right will always result in 0.
+
+                    Byte  | 00000001
+                    Int16 | 00000000 00000001
+                    Int32 | 00000000 00000000 00000000 00000001
+            #>
+
+            1 | Should -Be 3 -shr 1
+            __ | Should -Be 7 -shr 1
+        }
+
+        It 'can be used to create larger values from a sequence of bytes' {
+            <#
+                A 32-bit integer is made up of four bytes.
+
+                The bytes are shifted to the left based on their position. The most significant
+                byte is shifted 24 bits to the left (3 * 8).
+
+                The most significant byte is first, the bytes below are in Big Endian order.
+
+
+                    Binary            | 00010110  | 00111000 | 11000000 | 00001111 |
+                    ------------------+-----------+----------+----------+----------+
+                    Byte Value        | 22        | 56       | 192      | 15       |
+                    ------------------+-----------+----------+----------+----------+
+                    Bitwise operation | -shl 24   | -shl 16  | -shl 8   |          |
+                    ------------------+----------=+----------+----------+----------+
+                    Int32 Value       | 369098752 | 3670016  | 49152    | 15       |
+
+                The final Int32 value, 372817935, is obtained by adding each of the individual pieces together.
+            #>
+
+            # Hint: set one bit for each value
+            $Bytes = @(
+                '________'
+                '________'
+                '________'
+                '________'
+            ) | ConvertFrom-Binary
+
+            [Int]$Bytes[0] -shl 24 | Should -Be 1073741824
+            [Int]$Bytes[1] -shl 16 | Should -Be 2097152
+            [Int]$Bytes[2] -shl 8 | Should -Be 4096
+            $Bytes[3] | Should -Be 8
+
+            $Value = (
+                ([Int]$Bytes[0] -shl 24) +
+                ([Int]$Bytes[1] -shl 16) +
+                ([Int]$Bytes[2] -shl 8) +
+                $Bytes[3]
+            )
+
+            $Value | Should -Be 1652126821
+            $Bytes -join ' ' | ConvertFrom-Binary -To Int32 | Should -Be $Value
+        }
+    }
+
+
+    Context 'About the signing bit' {
+        <#
+            A byte is an example of an unsigned integer. The values can range from 0 to 255, the value cannot
+            be a negative number.
+
+            Signed integers reserve a single bit to denote whether the value is positive or negative.
+
+            The signing bit takes the place of the most significant byte.
```

Comment:
Probably also worth explicitly mentioning which value in the signing bit indicates positive / negative here.